index.html

```diff
@@ -5,22 +5,113 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Homework Assignment 3</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,800" rel="stylesheet">
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1>Relaxr</h1>  
+				<nav class="alignright clearfix"> 
+					<ul class="clearfix"> 
+						<li>	
+							<a href="#">About</a>
+						</li>
+						<li>
+							<a href="#">FAQ</a>
+						</li>
+						<li>
+							<a href="#">Team</a>
+						</li>
+						<li>
+							<a href="#">Contact Us</a>
+						</li>
+						<li>
+							<a href="#">Blog</a> 
+						</li>
+					</ul>	 
+				</nav>
 			</header>
+
+			<div class="container clearfix">
+				<div class="main alignleft">
+					<div class="article">
```

Comment:
no problem using the `"article"` class, however I recommend using the `<article></article>` element as well instead of a `<div></div>`